Factory/HttpFoundationFactory.php

```diff
@@ -140,11 +154,12 @@ protected function getTemporaryPath()
      */
     public function createResponse(ResponseInterface $psrResponse)
     {
-        $response = new Response(
-            $psrResponse->getBody()->__toString(),
+        $response = new StreamedResponse(
```

Comment:
I don't think we should use stream responses in every possible cases because `StreamedResponse` have some downsides like preventing the debug toolbar from being injected.

Can this be made configurable ?